code/modules/mob/living/silicon/emote.dm

```diff
@@ -0,0 +1,116 @@
+/datum/emote/silicon
+	mob_type_allowed_typecache = list(/mob/living/silicon/robot)
+	emote_type = EMOTE_AUDIBLE
+	keybind_category = CATEGORY_ROBOTIC_EMOTE
+
+/datum/emote/silicon/salute
+	key = "salute"
+	key_third_person = "salutes"
+	message = "salutes."
+	message_param = "salutes at %t."
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/silicon/clap
+	key = "clap"
+	message = "claps."
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/silicon/flap
```

Comment:
Same

Reply:
god knows man this was how they used to be. who even knows why they have wings